README.md

```diff
@@ -21,11 +21,27 @@ You can find the installation documentation for the
 The documentation for advanced usage of Jupyter notebook can be found
 [here](https://jupyter-server.readthedocs.io/en/latest/).
 
-For a local installation, make sure you have
+To install the latest release locally, make sure you have
 [pip installed](https://pip.readthedocs.io/en/stable/installing/) and run:
 
     $ pip install jupyter_server
 
+### Versioning and Branches
+
+If Jupyter Server is a dependency of your project/application, it is important that you pin it to a version that works for your application. Currently, Jupyter Server only has minor versions and patch versions. Different minor versions may include API-changes while patch versions do not change API.
+
+Each minor version has its own branch in this repo, and patches are applied to their relevant minor version. The master branch of the latest minor version.
```

Comment:
Should this be more like: "The master branch _represents_ the latest minor version."

Reply:
:+1: